cinema/views.py

```diff
@@ -34,6 +36,18 @@
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
+        strings = ["actors", "genres", "title"]
+        for string in strings:
+            string = self.request.query_params.get(string)
+            if string:
+                string_ids = [int(str_id) for str_id in string.split(",")]
+                queryset = queryset.filter(id__in=string_ids).all()
```

Comment:
The filtering logic for `actors` and `genres` is incorrect. The current implementation filters by `id`, which does not correspond to the intended fields. You should filter by `actors__id__in=string_ids` or `genres__id__in=string_ids` instead.